skills_taxonomy_v2/pipeline/skills_extraction/skills_naming_utils.py

```diff
@@ -20,69 +22,134 @@
 from skills_taxonomy_v2.pipeline.sentence_classifier.sentence_classifier import (
     BertVectorizer,
 )
+from skills_taxonomy_v2.getters.s3_data import load_s3_data, save_to_s3
+from skills_taxonomy_v2 import BUCKET_NAME
+
+from pattern.text.en import singularize
+from collections import OrderedDict
+
+from nltk.corpus import stopwords
+
+nltk.download("stopwords")
 
 logger = logging.getLogger(__name__)
 
 nltk.download("wordnet")
 
 
+def get_new_skills_embeds(new_skills_embeds_path, bucket_name):
+    """
+    takes as input string path to new skills embeddings directory and bucket name.
+    outputs:
+        dictionary of new skills embeddings.
+    """
+
+    s3 = boto3.resource("s3")
+
+    sentence_embeds = {}
+    all_sentence_embeds = []
+
+    for i in tqdm(range(0, 8)):  # 8 files
+
+        new_sentences_dict = load_s3_data(
+            s3, bucket_name, new_skills_embeds_path + f"sentences_data_{i}.json"
+        )
+        all_sentence_embeds.append(new_sentences_dict)
+
+    # https://stackoverflow.com/questions/57340332/how-do-you-combine-lists-of-multiple-dictionaries-in-python
+    for k in all_sentence_embeds[0].keys():
+        sentence_embeds[k] = sum(
+            [skills_dict[k] for skills_dict in all_sentence_embeds], []
+        )
+
+    return sentence_embeds
+
+
 def replace_ngrams(sentence, ngram_words):
     for word_list in ngram_words:
         sentence = sentence.replace(" ".join(word_list), "-".join(word_list))
     return sentence
 
 
-def get_top_tf_idf_words(clusters_vect, feature_names, top_n=2):
-    """
-    From https://stackoverflow.com/questions/34232190/scikit-learn-tfidfvectorizer-how-to-get-top-n-terms-with-highest-tf-idf-score
-    """
-    sorted_nzs = np.argsort(clusters_vect.data)[: -(top_n + 1) : -1]
-    return feature_names[clusters_vect.indices[sorted_nzs]]
-
-
-def clean_cluster_descriptions(sentences_data):
+def clean_cluster_description(sentences, cluster_number):
```

Comment:
I don't think the `clean_cluster_description` or `get_clean_ngrams` functions need `cluster_number` as an input at all since they only process one skill as a time anyway.